website/partners/api/v1/urls.py

```diff
@@ -3,6 +3,5 @@
 from . import viewsets
 
 router = routers.SimpleRouter()
-router.register(r"partners/events", viewsets.PartnerEventViewset)
```

Comment:
I don't think we should change v1 api endpoints at this point. Not that many people use this, but if we deprecate v1 to change it in v2, why change the old urls?

Reply:
Ah yes I forgot that I changed this and had to solve it somehow, I can definitely not remove this I think. App is using it.

Reply:
Fixed! Should not change now.

I also had to address the v2 API since the app has been reviewed by apple. But I have changed those to be redirects to the new path.